src/main/java/org/jspecify/nullness/NullMarked.java

```diff
@@ -25,7 +25,7 @@
 
 /**
  * Indicates that within the annotated scope (class or package), type usages <i>generally</i> do
- * <i>not</i> include {@code null} as a value, unless they are individually marked otherwise.
+ * <i>not</i> include {@code null} as a value, unless they are individually marked otherwise using {@link Nullable}.
  * Without this annotation, these type usages would instead have <i>unspecified nullness</i>.
```

Comment:
```suggestion
 * <i>not</i> include {@code null} as a value, unless they are individually marked otherwise using {@link Nullable} (or checker-specific other annotations).
 * Without this annotation, unmarked type usages would instead have <i>unspecified nullness</i>.
```